src/components/inputs/ui-select-extended/ui-select-extended.tsx

```diff
@@ -137,7 +146,7 @@ const UISelectExtended: React.FC<OHRIFormFieldProps> = ({ question, handler, onC
             : field.value
         }
         conceptName={conceptName}
-        isInline
+        isInline={isInline}
```

Comment:
Was this done by the linter? the change seems redundant

Reply:
The `isInline` prop was defaulting to true by not passing a value, the value `isInline` passed to the prop is defined in as a `useMemo` hook above.